### PR TITLE
fix(traffic_light): fix traffic_light_arbiter pipeline

### PR DESCRIPTION
**Pull request by 1222-takeshi**
## Description

Previously, arguments were not passed from traffic_light.launch.xml to traffic_light_arbiter.launch.xml, causing some unexpected behavior.
With the changes in this PR, that problem has been solved. The system now works as expected.
## Tests performed
Tested by AWSIM

## Notes for reviewers
none

## Interface changes
none

## Effects on system behavior
none

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by tkimura4** [comment]
cc. @0x126 @TomohitoAndo

**Review by yukkysaito — APPROVED** [review]
LGTM

File: launch/tier4_perception_launch/launch/traffic_light_recognition/traffic_light.launch.xml [hunk 1]
```diff
@@ -9,7 +9,8 @@
   <arg name="classifier_model_path" default="$(find-pkg-share traffic_light_classifier)/data/traffic_light_classifier_mobilenetv2_batch_6.onnx" description="classifier onnx model path"/>
   <arg name="input/cloud" default="/sensing/lidar/top/pointcloud_raw" description="point cloud for occlusion prediction"/>
   <arg name="internal/traffic_signals" default="/perception/traffic_light_recognition/internal/traffic_signals"/>
-  <arg name="output/traffic_signals" default="/perception/traffic_light_recognition/traffic_signals"/>
+  <arg name="external/traffic_signals" default="/perception/traffic_light_recognition/external/traffic_signals"/>
+  <arg name="output/traffic_signals" default="/perception/traffic_light_arbiter/traffic_signals"/>
```
Comment (miursh) [inline launch/tier4_perception_launch/launch/traffic_light_recognition/traffic_light.launch.xml:13]: [Q]
Which node request "/perception/traffic_light_arbiter/traffic_signals" as output of perception? This change (output I/F of perception) doesn't sound correct to me
Which node publish "/perception/traffic_light_recognition/external/traffic_signals"?

Comment (1222-takeshi) [reply launch/tier4_perception_launch/launch/traffic_light_recognition/traffic_light.launch.xml]: @miursh 
I have corrected the output topic name as per your suggestion. :bow: 
https://github.com/autowarefoundation/autoware.universe/pull/4393/commits/308a641b6e1c9a061b1cc96dc8cfac43d80d1a7e

As for external/traffic_signals, I just created an argument for it. There was no major change intended.

Comment (1222-takeshi) [reply launch/tier4_perception_launch/launch/traffic_light_recognition/traffic_light.launch.xml]: I will create a separate PR to change the input topic of the behavior_velocity module.

**Review by miursh — APPROVED** [review]
Thank you for your help!
LGTM

**Review by knzo25 — APPROVED** [review]
Sorry for the delay. LGTM!

**Pull request merged**